tools/WinObjC.Packaging/WinObjC.Packaging.nuproj

```diff
@@ -17,22 +17,42 @@
   <PropertyGroup Label="Globals">
     <OutputPath>$(SolutionDir)\OutputPackages\$(Configuration)\</OutputPath>
     <ProjectGuid>ddc2aa5f-c8ae-48cb-adb7-9278985c3db7</ProjectGuid>
+    <SkipGetPackageContentsForOtherPlatforms>true</SkipGetPackageContentsForOtherPlatforms>
   </PropertyGroup>
   <PropertyGroup Label="Configuration" />
   <ItemGroup>
-    <None Include="Package.Default.props" />
-    <None Include="Package.Default.targets" />
-    <None Include="Package.props" />
-    <None Include="Package.targets" />
-    <None Include="WinObjC.Packager.props" />
-    <None Include="WinObjC.Packager.targets" />
-    <None Include="WinObjC.Packageable.props" />
-    <None Include="WinObjC.Packageable.targets" />
-    <None Include="project.json" />
+    <None Include="Package.Default.props" >
+      <IncludeInPackage>true</IncludeInPackage>
+    </None>
+    <None Include="Package.Default.targets" >
+      <IncludeInPackage>true</IncludeInPackage>
+    </None>
+    <None Include="Package.props" >
+      <IncludeInPackage>true</IncludeInPackage>
+    </None>
+    <None Include="Package.targets" >
+      <IncludeInPackage>true</IncludeInPackage>
+    </None>
+    <None Include="WinObjC.Packager.props" >
+      <IncludeInPackage>true</IncludeInPackage>
+    </None>
+    <None Include="WinObjC.Packager.targets" >
+      <IncludeInPackage>true</IncludeInPackage>
+    </None>
+    <None Include="WinObjC.Packageable.props" >
+      <IncludeInPackage>true</IncludeInPackage>
+    </None>
+    <None Include="WinObjC.Packageable.targets" >
+      <IncludeInPackage>true</IncludeInPackage>
```

Comment:
><IncludeInPackage>true</IncludeInPackage> [](start = 6, length = 41)

would be a little cleaner to maybe change the itemdefinitiongroup for None and only manually specify the ones that **shouldn't** be in the package